src/time/estimate.zig

```diff
@@ -61,3 +61,61 @@ pub fn printTimeEstimate(
         });
     }
 }
+
+pub fn printTimeEstimateStd(
```

Comment:
just a small nit, could we name this to be `printTimeEstimateStderr` instead.

Reply:
[a535dd0](https://github.com/Syndica/sig/pull/279/commits/a535dd007aec80cee870207228b23aa83b8b4d1e)